frontend/src/pages/team/Devices/dialogs/DeviceCredentialsDialog.vue

```diff
@@ -15,14 +15,14 @@
                     <template v-if="otc">
                         <label class="block font-bold mb-2">Install Device Agent</label>
                         <p>Run this command on the hardware where you want your Remote Instance to run:</p>
-                        <pre class="overflow-auto text-xs font-light p-4 my-2 border rounded bg-gray-800 text-gray-200">npm install @flowfuse/device-agent</pre>
+                        <pre class="overflow-auto text-xs font-light p-4 my-2 border rounded bg-gray-800 text-gray-200">sudo npm install -g @flowfuse/device-agent</pre>
                         <div class="flex flex-row justify-end space-x-2 -mt-1">
-                            <ff-button kind="tertiary" size="small" @click="copy('npm install @flowfuse/device-agent')">
+                            <ff-button kind="tertiary" size="small" @click="copy('sudo npm install -g @flowfuse/device-agent')">
```

Comment:
As noted in the original issue, the instructions will be slightly different for Windows users - they don't have `sudo`.

Can we have a tabbed UI for the different OS options - the precise content for the Windows option will need to come from @Steve-Mcl